src/Silverback.Integration.Kafka/Messaging/Messages/KafkaKeyHelper.cs

```diff
@@ -1,44 +1,86 @@
-﻿// Copyright (c) 2020 Sergio Aquilini
-// This code is licensed under MIT license (see LICENSE file for details)
```

Comment:
I'm not sure what line endings you really want to use.
In editor config is CRLF but I'm unsure if this is consistent across the files.

Reply:
I use CRLF. But the GIT client on windows by default converts them to LF when committing and that's just how I use it (`core.autocrlf true`). So in the repo they are actually ending with LF.